src/boilerplate/orchestration/javascript/raw/boilerplate-generator.ts

```diff
@@ -251,6 +280,7 @@ class BoilerplateGenerator {
         `\nimport fs from 'fs';
         \n`,
         `\nimport { getContractInstance, registerKey, getInputCommitments } from './common/contract.mjs';`,
+        `\nimport  joinCommitments  from './common/contract.mjs';`,
```

Comment:
Just for cleanliness, we could join together these two lines (if the imports are unused, that's fine):
```suggestion
        `\nimport { getContractInstance, registerKey, getInputCommitments, joinCommitments } from './common/contract.mjs';`,
```

Reply:
thanks for the suggestion.. combined the import call